### ISSUE TITLE
Insert code for a new class

### ISSUE THREAD

**Issue by gonetoofar** [BowlerHatLLC/vscode-as3mxml#413]
In Flashbuilder, when you create a new class it puts in some boilerplate code. This can't be done in vscode (unless I'm missing something).
One option is to add the following snippet to the snippets file (distribution/src/assembly/snippets/actionscript.json):

	"Package Definition": {
		"prefix": "package",
		"body": [
			"package",
			"{",
			"\tpublic class ${TM_FILENAME_BASE}",
			"\t{",
			"\t\tpublic function ${TM_FILENAME_BASE}(${1:parameters})",
			"\t\t{",
			"\t\t\t$0",
			"\t\t}",
			"\t}",
			"}"
		],
		"description": "Package Definition"
	}

Also, most of the snippets are discoverable but I didn't know about ctor and class until I found this file.

**Comment by joshtynjala** [comment]
When you create a new *.as* file, just start typing `package`, and it will give you a completion item that adds the package block, including the full package name and curly braces. So, for example, if you created the file *com/example/MyClass.as*, you can easily complete the package block like this:

```actionscript
package com.example
{
}
```

Then, you can use the "class" or "interface" snippet to add the one that you need inside the package block. It will automatically fill in the class/interface name based on the filename. So, if you start typing "class" inside the package block, you can get this:

```actionscript
package com.example
{
	public class Example()
	{
		public function Example(parameters)
		{
		}
	}
}
```

So, basically, it's two steps instead of one.

I have no plans to add more snippets at this time. I provide the same snippets as TypeScript, and I plan to keep them in sync with Microsoft's defaults.

VSCode supports custom snippets, though, so you are free to add as many as you want in your local environment.
